internal/core/metadata/rest_deviceprofile.go

```diff
@@ -594,15 +586,15 @@ func notifyProfileAssociates(
 		return err
 	}
 
-	// Get the services for each device
+	// Get the services for each dev
 	// Use map as a Set
 	dsMap := map[string]models.DeviceService{}
 	ds := []models.DeviceService{}
-	for _, device := range d {
+	for _, dev := range d {
 		// Only add if not there
-		if _, ok := dsMap[device.Service.Id]; !ok {
-			dsMap[device.Service.Id] = device.Service
-			ds = append(ds, device.Service)
+		if _, ok := dsMap[dev.Service.Id]; !ok {
+			dsMap[dev.Service.Id] = dev.Service
+			ds = append(ds, dev.Service)
```

Comment:
Looks like a unintended search/replace change.  If not, why?

Reply:
There was an IDE warning about `device` overriding a package name.